### ISSUE TITLE
markets should only have 2 options to (scalar long/short) (binary: yes/no) not 4 to buy/sell long/short

### ISSUE THREAD

**Issue by kermit4** [protofire/omen-exchange#1857]
I can't think of any reason someone would want to hold opposing positions, so presenting the option to makes the user think they don't understand the system as it implies there's a reason they'd want to.

Or maybe I still don't understand, myself, despite working on it for weeks.

**Comment by KadenZipfel** [comment]
Maybe I'm misunderstanding, but the reason there's the option to buy and sell is because you can actually sell shares that you've purchased back to the AMM for a different price from what you paid (since the price changes as the proportion of each outcome held by the AMM changes). So selling long shares is actually a different action from buying short shares.

**Comment by kermit4** [comment]
Right, so if you had long shares, would you ever want to buy short shares?

If not, the option shouldn't be presented.  It should be just two options, to long/short.

@sacshin (Nathan/Arhat) explained it as:

"I like predictit and it is fine to buy yes or buy no. But when you have buy and sell paired with short and long it becomes redundant. Buying short is like selling long, and selling short would imply buying long. One selling short would second guess themselves, are they selling and thus shorting, or are they selling the short, thus longing. :D"

**Comment by kermit4** [comment]
@sacshin suggested a screenshot of the current UI for reference:

![image](https://user-images.githubusercontent.com/565572/116480421-ef3f1d00-a846-11eb-93a8-fe9034ff2c92.png)
